app/models/course.rb

```diff
@@ -257,10 +259,15 @@ def set_defaults
     self.start_at ||= Time.zone.now.beginning_of_hour
     self.end_at ||= self.start_at + 1.month
     self.default_reference_timeline ||= reference_timelines.new(default: true)
+    self.default_timeline_algorithm ||= 0 # 'fixed' algorithm
 
     return unless creator && course_users.empty?
 
-    course_users.build(user: creator, role: :owner, creator: creator, updater: updater)
+    course_users.build(user: creator,
+                       role: :owner,
+                       creator: creator,
+                       updater: updater,
+                       timeline_algorithm: default_timeline_algorithm)
```

Comment:
No need to specify this here as this course_user is an owner of the course and not student.

Reply:
I see, thanks!